public/plugin.tsx

```diff
@@ -227,6 +231,36 @@ export class AssistantPlugin
           }
         },
       });
+
+      core.application.register({
```

Comment:
Could you add a feature flag for this feature? You can take `text2viz` as example.